test/storage/file.js

```diff
@@ -370,6 +370,30 @@ describe('File', function() {
   });
 
   describe('createReadStream', function() {
+
+    function getFakeRequest(data, fakeResponse) {
+      function FakeRequest(req) {
+        if (!(this instanceof FakeRequest)) {
+          return new FakeRequest(req);
+        }
+
+        var that = this;
+
+        stream.Readable.call(this);
+        this._read = function() {
+          this.push(data);
+          this.push(null);
+        };
+
+        setImmediate(function() {
+          that.emit('response', fakeResponse);
+          that.emit('complete', fakeResponse);
```

Comment:
We should probably remove response from the `complete` at sometime then you think?